samples/highcharts/demo/areaspline/demo.js

```diff
@@ -1,45 +1,64 @@
+// Retrieved from https://www.ssb.no/jord-skog-jakt-og-fiskeri/jakt
 Highcharts.chart('container', {
     chart: {
         type: 'areaspline'
     },
     title: {
-        text: 'Average fruit consumption during one week'
+        text: 'Moose and deer hunting in Norway, 2000 - 2021'
+    },
+    subtitle: {
+        align: 'center',
+        text: 'Source: <a href="https://www.ssb.no/jord-skog-jakt-og-fiskeri/jakt" target="_blank">SSB</a>'
     },
     legend: {
         layout: 'vertical',
         align: 'left',
         verticalAlign: 'top',
-        x: 150,
-        y: 100,
+        x: 120,
+        y: 70,
         floating: true,
         borderWidth: 1,
         backgroundColor:
             Highcharts.defaultOptions.legend.backgroundColor || '#FFFFFF'
     },
     xAxis: {
         categories: [
```

Comment:
The number of categories makes the X axis cluttered, plus the nature of this axis is numerical/temporal. I would use a numerical axis in this case, and instead let the tooltip say something like `Hunting season starting autumn 2020` to indicate that it spans over two calendar years?